src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body
+    class="card" 
+    data-qa="card"
+  >
+    <img 
+      src="images/imac.jpeg" 
+      alt="Apple iMac27"
+      class="imac card__imac"
+    >
+
+    <h1 class="title card__title">
+      APPLE A1419 iMac 27" Retina<br> 
+      5K Monoblock (MNED2UA/A)
+    </h1>
+
+    <p class="product-code card__product-code">
+      Product code: 195434
+    </p>
+
+    <div class="stars">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__reviews">Reviews: 5</div>
+    </div>
+
+    <div class="price card__price">
+      <p class="price-text">Price:</p>
+      <p class="price-value">$2,199</p>
+    </div>
+
```

Comment:
![image](https://user-images.githubusercontent.com/70009205/209988870-f218b232-e008-433d-99dd-0c95f3d112b2.png)


```suggestion
    <div class="card__price-container">
      <p class="card__price-text">Price:</p>
      <p class="card__price-value">$2,199</p>
    </div>

```